docker-compose/docker-compose-certify/docker-compose.yml

```diff
@@ -70,7 +70,7 @@ services:
     networks:
       - network
   esignet:
-    image: 'mosipid/esignet:1.4.0'
+    image: 'mosipid/esignet:1.4.1'
```

Comment:
More changes:

1. Artifactory container will need to add file `esignet-image.zip`
2. Right now the testing of mock has been done using `mosipdev/artifactory-server:release-0.9.x-INJI`, this change would have to be done for local setup and the image should come from `mosipid/*` repo

Reply:
Done. Requesting images to be moved on `mosipqa`.